src/Symfony/Bridge/Twig/Tests/Extension/TranslationExtensionTest.php

```diff
@@ -283,6 +283,6 @@ protected function getTemplate($template, $translator = null)
         $twig = new Environment($loader, ['debug' => true, 'cache' => false]);
         $twig->addExtension(new TranslationExtension($translator));
 
-        return $twig->loadTemplate('index');
+        return $twig->load('index')->unwrap();
```

Comment:
we should not need to unwrap here. We only call `render()`, which is available on the non-internal TemplateWrapper API directly.

Reply:
fixed